### PR TITLE
fuel survey support for template

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?
Case service changes to support for quarterly fuels survey support
# How to test?

# Trello
https://trello.com/c/mzt8q6oH/1397-quarterly-fuels-survey-multiple-template-support

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -143,13 +144,13 @@ retries:
   backoff: 5000
 
 gcp:
-  project: ras-rm-sandbox
-  receiptSubscription: ras-rm-receipt
-  caseNotificationTopic: "test_topic"
-  caseNotificationSubscription: "test_subscription"
+  project: ras-rm-dev
+  receiptSubscription: sdx-receipt-sinhaa
```
Comment (insacuri) [inline src/main/resources/application.yml:148]: Probably wanna revert these before you merge it in

**Review by insacuri — CHANGES_REQUESTED** [review]
Code looks good.  Just need to revert the config changes and it's good to go

File: src/main/resources/application.yml (hunk 1 above)
Comment (creativeappdevamit) [reply src/main/resources/application.yml:148]: ahh... sorry for this.. will revert it

**Pull request merged**